src/rkopenmdao/runge_kutta_integrator.py

```diff
@@ -1211,6 +1250,9 @@ def _add_functional_perturbations_to_state_perturbations(
     def _run_step_jacvec_rev(
         self, serialized_state, serialized_state_perturbations
     ):
+        """
+        Function to get the sensitvities of the rkschemes at the last step.
```

Comment:
I would add something along the lines of "of the current checkpointing iteration" to the end of the sentence.